### PR TITLE
add apply destination rule section for traffic management

### PR DESCRIPTION
**Pull request by irisdingbj**
Many users are having issues when trying istio traffic management guide as we have not added the destination rule part: eg: https://github.com/istio/istio/issues/8660,

this PR is used for adding this info.

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit c6c516f91eb461b6581ae865ec36752ec901cf36

https://app.netlify.com/sites/preliminary-istio/deploys/5b99ce914ed62f753e753685

**Comment by irisdingbj** [comment]
@frankbu add a Note part as your suggestion

File: content/docs/tasks/traffic-management/request-routing/index.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ In this case, the virtual services will route all traffic to `v1` of each micros
     Because configuration propagation is eventually consistent, wait a few seconds
     for the virtual services to take effect.
 
+    __NOTE__: If you haven't already applied destination rules, follow the [Apply default destination rules](/docs/examples/bookinfo/#apply-default-destination-rules).
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/request-routing/index.md]: add "instructions in" after "follow the" and make link title all caps: 
... follow the instructions in [Apply Default Destination Rules](/docs/...

Also please move the note up, before the first instruction.

File: content/docs/tasks/traffic-management/request-routing/index.md [hunk 2]
```diff
@@ -45,6 +45,8 @@ In this case, the virtual services will route all traffic to `v1` of each micros
     Because configuration propagation is eventually consistent, wait a few seconds
     for the virtual services to take effect.
 
+    {{< idea_icon >}} If you haven't already applied destination rules, follow follow the instructions in [Apply Default Destination Rules](/docs/examples/bookinfo/#apply-default-destination-rules).
```
Comment (sdake) [inline content/docs/tasks/traffic-management/request-routing/index.md]: follow used twice.  I think `>` would be preferred to an `{{< idea_icon >}}`.  Generally we escalate in the docs bold, and then with `>` and finally emojis `{{< idea icon >}}`.

Comment (frankbu) [reply content/docs/tasks/traffic-management/request-routing/index.md]: @irisdingbj can you please remove the extra "follow"

**Comment by frankbu** [comment]
Closing in favor of #2801

**Pull request closed without merging**